dist/valid-head.js

```diff
@@ -0,0 +1,23 @@
+// Used for WPT
+// [valid-head]
+// Use DOMParser to extract head elements so browser can't truncate the head
+// Loop through each node and check against valid elements array
+// Returns object with ARRAY and BOOL
+var html = $WPT_BODIES[0].response_body;
+var parser = new DOMParser();
+var xmlDoc = parser.parseFromString(html,"text/xml");
```

Comment:
Looks like this might throw a `TypeError`. Could we add try/catch handling?

Reply:
Done! Sorry for the wait